microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -58,18 +63,23 @@ Use the `getDigitalGoodsService` method to connect to the Microsoft Store Billin
 The Digital Goods API was designed to be compatible with various browsers and digital stores, similar to how the Payment Request API is browser-agnostic and can be used with different payment providers.  To retrieve an instance of the service for Microsoft Store Billing, pass the string `"https://store.microsoft.com/billing"` as the payment method to the `getDigitalGoodsService` method.
 
 If the method throws an error, the Microsoft Store Billing payment method is not available (such as when the user is accessing your PWA through the browser).  Alternatively, consider providing a different payment method for transactions.
```

Comment:
expand on the reasons that might make the payment method not available.
when is the method available exactly?
what are the requirements for it to be available - PWA installed from the store?
are there any other required settings?

@runyuany

Reply:
The payment method will be available only for Edge based PWA which installed from Microsoft Store, on Windows OS. No other required settings needed.